### DISCUSSION TITLE
Support for Android Junit5

### DISCUSSION THREAD

**Discussion by gopidholakia** [Banno/Gordon#123] [Ideas]
Gordon can only support Junit4, it would be great if I get support for JUnit5 as well.

[android-junit5](https://github.com/mannodermaus/android-junit5)

what I have did to have the suppoert for JUnit5 is, edit TestSuiteLoader

**From**
`private val Annotatable.isTestMethod
    get() = annotationNames.any { it == "org.junit.Test" }`
    
**To**
`private val Annotatable.isTestMethod
    get() = annotationNames.any { it in setOf("org.junit.Test", "org.junit.jupiter.api.Test") }`
    
 This change is working for me, and It would be better if I get support by Gordon itself.

**Comment by joshschriever** [comment]
Closing the discussion to keep conversation on the issue #124
